### DISCUSSION TITLE
Should we allow users to supply own font?

### DISCUSSION THREAD

**Discussion by karrui** [opengovsg/design-system#220] [Ideas]
Currently, our components and text styles are styled with the `Inter` font in mind. Should we allow users to use any fonts they like? 

This can be achieved by writing `textStyles` with a tokenized `fontFamily`, so all users need to do (at a minimum, without replacing the whole textStyle object) is to override theme's global fonts.

But MAYBE we want to standardize that all OGP design usage should be with the Inter font in mind, so some more ruminating is required

**Comment by seaerchin** [comment]
But if the product override the default font (inter), they're doing it deliberately so it's more of a conscious choice isn't it? 

I think should, so that the design system will have a wider use case beyond ogp hehe
